src/engine/rust.rs

```diff
@@ -62,14 +63,27 @@ struct Miner {
 }
 impl Miner {
 	const BASE_BYTES: f64 = 10.5;
+	const BROADCAST_SLEEP_SECONDS: u32 = 15;
 	const INPUT_BYTES_BASE: f64 = 57.5;
+	const MAX_BROADCAST_NUM: u32 = 20;
+	const MAX_SEQUENCE: u32 = u32::MAX;
+	// OP_RETURN size
+	// 8-bytes value(roughly estimate), a one-byte script’s size
+	// actual value size depends precisely on final nonce
+	const OP_RETURN_BYTES: f64 = 21. + 8. + 1.;
```

Comment:
I'm curious about how you obtained these values.

Reply:
Hi Aurevior, my understanding is:
<8 bytes value/amount> + <1 byte len of following script> + <1 byte OP_RETURN(0x6a)> + <1 byte len indicating the rest> + <10 byte unix_timestamp> + <1 byte for colon separator(:)> + <8 bytes nonce, roughly estimation>
so: 8 + 1 + 1 + 1 + 10 + 1 + 8 = 30 bytes

Reply:
so the comments maybe a little confusing: ```8-bytes nonce``` rather than ```8-bytes value``` and ```value size``` should be ```nonce size``` @AurevoirXavier